README.md

```diff
@@ -35,6 +35,45 @@ Users should be able to deploy the bridge into their cluster with a provider sel
 `ENABLE_PROVIDER_LOG` To enable full provider logs
 `SKIP_CRD_CREATION` To skip CRD creation at startup
 
+## Provider Config
+
+There are two ways to configure the provider used by the operator. 
+
+1. Normal provider environment variables
+2. An environment variable which contains the HCL for provider config
```

Comment:
It could be nice to also support a ConfigMap - what do you think? Maybe raise an issue for it?

Reply:
I gave this one a go but turned out horribly, added more context in the PR thread below